packages/search/src/components/actions-api/query.js

```diff
@@ -35,25 +35,11 @@ export const getCountsApiQuery = async() => {
   return toURLSearchParams(Object.assign(defaultCountsApiQuery(), params))
 }
 
-export const getSortParams = async(page={}) => {
-  const params = {s: await sortQuery(), ...page }
-
-  // return toURLSearchParams(Object.assign(defaultQuery(), params))
-}
-
 // private function
+function getMongoSortQuery() {
+  const sortPropertyName = readSearchParamsByName('s')[0]
 
-function sortQuery() {
-  const sortType = readParamType('s')
-  let s = {}
-
-  const key = Object.keys(sortType)
-  const type = 'meta'
-
-  // Unsure of how to form s to send to api here?
-
-
-  return s
+  return sortPropertyName? { [`meta.${sortPropertyName}`]: -1 }: { 'meta.modifiedOn': -1, 'meta.createdOn': -1 }
```

Comment:
@hmalik129 please review this line and if you have any questions on what is happening let me know.